lib/x11utils.pm

```diff
@@ -106,8 +106,14 @@ sub ensure_unlocked_desktop {
             record_info('Guest disabled display', 'Might be consequence of bsc#1168979');
         }
         if (match_has_tag('authentication-required-user-settings') || match_has_tag('authentication-required-modify-system')) {
-            type_password;
-            assert_and_click "authenticate";
+            wait_still_screen(2);
+            if (!check_screen([qw(authentication-required-user-settings authentication-required-modify-system)])) {
+                record_soft_failure "bsc#1168979 - screenbuffer not updated after screen is locked - Previous content of screen is leaked for a short period of time when waking up the display or switching back from textmode tty";
```

Comment:
let's break this line in two to improve readability.

Reply:
I think just calling the record soft failure with the bsc number should be enough

Reply:
Or I can just shorten the message